workflows/checks/check-md5.nf

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+// run parameters
+params.run_metafile = 's3://ccdl-scpca-data/sample_info/scpca-library-metadata.tsv'
+params.outdir = 's3://nextflow-ccdl-results/scpca-checks/'
+
+process check_md5{
+  container 'ubuntu:20.04'
+  publishDir "${params.outdir}"
+  input:
+    tuple val(id), val(md5_file), path(files)
+  output:
+    path outfile
+  script:
+    outfile = "${id}-md5check.txt"
+    """
+    md5sum -c ${md5_file} > ${outfile}
```

Comment:
Here is the step where the md5checksum is ran and output assigned for each file, correct?